library/CMakeLists.txt

```diff
@@ -74,7 +73,7 @@ target_include_directories(ic_proto
 )
 target_link_libraries(ic_proto
```

Comment:
First of all, thanks for your PR!

Secondly, I want to apologize. This codebase is in a transition and there's no way you could've known that. The logical "head" of this target is actually over [here](https://gitlab.com/jbt/ipfs_client/-/blob/ci/CMakeLists.txt?ref_type=heads#L71) at least for now while waiting on an answer about something else... 

If I can get my head around your use case and reproduce the issue then I'll move a version of your fix into the place I'll need things to be as I clean up my mess.

Reply:
my use case was just getting the following to work on ubuntu noble LTS:

```
cmake -D CMAKE_BUILD_TYPE=Release -D DOWNLOAD_CHROMIUM=TRUE -S ~/ipfs-chromium -B ~/build
```

Reply:
```
cmake --version
cmake version 3.28.3

python3 --version
Python 3.12.3

ldd --version
ldd (Ubuntu GLIBC 2.39-0ubuntu8.4) 2.39
```

Reply:
I appreciate the info. Your versions are reasonably close to my own, well within what I intended to support. And that cmake command was definitely intended to work.